ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderTest.java

```diff
@@ -464,6 +469,41 @@ public void shouldUseCorrectSerdeForUnwindowedKey() {
     );
   }
 
+  @Test
+  public void shouldNotContainSchemaIdInMaterializedSchema() {
+    when(keyFormatInfo.copyWithoutProperty("schemaId")).thenReturn(keyFormatInfoWithoutProps);
+    when(valueFormatInfo.copyWithoutProperty("schemaId")).thenReturn(valueFormatInfoWithoutProps);
+
+    // Given:
+    givenConsumed(consumed, keySerde);
+
+    tableSource = new TableSource(
+        new ExecutionStepPropertiesV1(ctx),
+        TOPIC_NAME,
+        formats,
+        TIMESTAMP_COLUMN,
+        SOURCE_SCHEMA,
+        SystemColumns.ROWPARTITION_ROWOFFSET_PSEUDOCOLUMN_VERSION,
+        formats
+    );
```

Comment:
Replace this with `givenUnwindowedSourceTable()`. The helper methods are there to de-dup code and make the tests easier to read.

Reply:
Ah.. I was testing something and I created this. I forgot to undo this change. Thanks for catching this.